core-api/src/main/java/org/silverpeas/core/date/AbstractDateTemporal.java

```diff
@@ -32,7 +32,9 @@
 /**
  * User: Yohann Chastagnier
  * Date: 06/12/13
+ * @deprecated Use the java.time API
  */
+@Deprecated
 public abstract class AbstractDateTemporal<T extends Temporal<? super T>> extends java.util.Date
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1133)